packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -64,7 +70,9 @@ export class EpochTracker {
         // Add epoch in fetch request.
         const request = this.addEpochInRequest(url, fetchOptions, addInBody);
         try {
-            const response = await fetchAndParseAsJSONHelper<T>(request.url, request.fetchOptions);
+            const response = await this.rateLimiter.schedule(
+                async () => fetchAndParseAsJSONHelper<T>(request.url, request.fetchOptions),
```

Comment:
I'd love to get some telemetry / statistics on usage. I do not know what kind, but something that can help us understand the impact. I.e. the max number of items in the queue waiting, or something along these lines.

Reply:
Also maybe total time until queue drained, but only if queue was not empty.
Here by queue I mean any requests that were throttled and waited for first 12 to be executed.

Reply:
Added telemetry for length of queue. Total time is already recorded by wrapper.